gno.land/pkg/gnoweb/frontend/css/tx.config.js

```diff
@@ -39,10 +41,22 @@ export default {
         600: "#54595D", // Content color
         800: "#131313", // Background dark color
         900: "#080809", // Title color
+        // dark mode grays
+        d50: "#121212",    // Softer than pure black for backgrounds
+        d100: "#2A2A2A",   // Subtle elevation color
+        d200: "#3D3D3D",   // Secondary elevation
+        d300: "#949494",   // Muted text (good)
+        d400: "#777777",   // Border color (good)
+        d600: "#ADADAD",   // Primary text - slightly higher contrast
+        d800: "#E0E0E0",   // High contrast text
+        d900: "#F5F5F5"    // Brightest text (headers)
       },
       green: {
         400: "#2D8D72", // Primary dark color
         600: "#226C57", // Primary light color
+        // dark mode  
```

Comment:
This is not a green but a white. This colors already exists in the palette: `light`.

Reply:
I can handle the palette updates myself if you grant me access to your repository. It might be quicker and easier than multiple back-and-forth iterations. 👍